plasTeX/Renderers/PageTemplate/simpletal/simpleTAL.py

```diff
@@ -841,10 +841,11 @@ def addTag (self, tag, tagProperties={}):
 					'singletonTag'    - A boolean to indicate that this is a singleton flag
 		"""
 		# Add the tag to the tagStack (list of tuples (tag, properties, useMacroLocation))
+                tagProperties = tagProperties or {}
```

Comment:
Maybe... the code in this listing looks like it's indented improperly.  But looking at the full file it looks fine.